README.md

```diff
@@ -100,40 +100,36 @@ https://arewecompressedyet.com/analyzer/?d=https://people.xiph.org/~mbebenita/an
 
 ## Coding style
 Check code formatting with [rustfmt](https://github.com/rust-lang-nursery/rustfmt) before submitting a PR.
-rav1e currently uses the nightly version of rustfmt.
+rav1e currently uses the stable version of rustfmt.
```

Comment:
Just replacing "nightly" with "stable" is a bit confusing, because the stable toolchain is not something unique. I guess it would be better, if the entire related text would be reworded in a better way.

For example, what does "currently" mean in this sentence? Are there any plans to switch to a non-stable toolchain? Why? This word produces too much questions. It would be better, if it wouldn't exist at all.

I think it would be better to replace this sentence with something like "rav1e uses the stable version of Rust (the `stable` toolchain)", and then continue with instructions without constantly repeating "stable" everywhere. Something like "to install `rustfmt`, execute...", and so forth.